app/kumactl/cmd/install/install_control_plane_test.go

```diff
@@ -139,16 +141,13 @@ var _ = Describe("kumactl install control-plane", func() {
 			ExpectMatchesGoldenFiles(actual, filepath.Join("testdata", given.goldenFile))
 		},
 		Entry("should generate Kubernetes resources with default settings", testCase{
-			extraArgs: []string{
-				"--without-kubernetes-connection",
```

Comment:
why are we changing this?

Reply:
I'm now applying it to _all_ the Entries. These tests run without an actual Kubernetes API server so I think it only makes sense to have this flag everywhere here

Reply:
Of course the actual motivator here is that with this PR we know attempt to query the API server if this flag isn't present. so some of these tests that didn't have the flag before now fail